lib/cli.rb

```diff
@@ -349,13 +349,14 @@ def set_priorities
 EOT
   option 'board-id', desc: 'Id of the board', required: true
   option 'target-board-id', desc: 'Id of the target board', required: true
+  option 'labelled', desc: 'Set true to label cards as - in the the last sprint', required: false, type: :boolean, default: false
```

Comment:
`the` in the description is twice. And it is not really clear, what about this for the desc: `If true cards are label as "In last sprint"`